### ISSUE TITLE
Gisto Windows - how to change the GitHub login?

### ISSUE THREAD

**Issue by ajit555** [Gisto/Gisto#301]
I logged using one GitHub account. Now I want to change the login and log using another GitHub account. It logs in automatically using first GitHub account and there is no option to change the account to second one.

**Comment by sanusart** [comment]
@ajit555 Log out from GitHub on the browser. Then it will be forced to ask for user/pass. This is how all Single Sign On works.

**Comment by ajit555** [comment]
That did not work. I logged out from GitHub on the Chrome browser. Still Gisto was logged in.  

In chrome browser, I have multiple profile setup and there are multiple GitHub accounts logged in each profile.

I had to uninstall the application and then manually delete all files and folder from "C:\Users\ajit5\AppData" folder and then re-install the application, then it allowed me to log in using another GitHub user account. But I am not sure if I can switch the account without doing this whole process again.

![Gisto-Windows](https://user-images.githubusercontent.com/194380/97436672-5e938d00-1948-11eb-8194-6b4e063a24fa.png)

In comparison, when I logged out of Lepton, it promptly asked me to log in using a GitHub account instead of auto-login even though the GitHub account was already logged in the Chrome browser. This behavior is expected in Gisto too.

**Comment by sanusart** [comment]
Yes, totally agree, log-in in general need to be revisited, GitHub deprecating user/pass login (aka Basic authentication) for apps.

**Comment by sanusart** [comment]
Closing as new rewritten version upcoming 

Ref: https://github.com/Gisto/Gisto/issues/387
